phpunit.xml.dist

```diff
@@ -13,10 +13,6 @@
     </testsuites>
     <filter>
         <whitelist>
-            <exclude>
-                <directory>./src/Command</directory>
```

Comment:
None of these directories exist, so there is no need for them to be excluded from the whitelist for collecting code coverage.

Reply:
I have moved all commands to `./src/Console/Command`  :grin: